### PR TITLE
update CHANGES

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
Include advice about checking repo/archive consistency?

File: docs/changes.rst [hunk 1]
```diff
@@ -1,6 +1,144 @@
...
+
+Version 1.0.4 (not released yet)
+--------------------------------
+
+New features:
+
+- borg serve --append-only, #1168
+  This was included because it was a simple change (append-only functionality
+  was already present via repository config file) and makes better security now
+  practically usable.
```
Comment (enkore) [inline docs/changes.rst]: , via SSH forced commands, see the documentation for details.

File: docs/changes.rst [hunk 2]
```diff
@@ -1,6 +1,144 @@
...
+- make borg build/work on OpenSSL 1.0 and 1.1, #1187
+  This was included so it builds on most recent OS distributions also.
```
Comment (enkore) [inline docs/changes.rst]: Move this to the "other changes" section?

File: docs/changes.rst (hunk 1 above)
Comment (ThomasWaldmann) [inline docs/changes.rst]: well, I assume that people consult the docs for more details (without saying that for many items).

Comment (enkore) [inline docs/changes.rst]: True

File: docs/changes.rst (hunk 2 above)
Comment (ThomasWaldmann) [inline docs/changes.rst]: hmm, yes, maybe not interesting for most people.

**Pull request merged**